packages/json-rpc/legacy-provider/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 0.2.4 - 2025-08-29
```

Comment:
you're missing the package version bump if you want to release it straight away (and also we're in the 30th now :P)